contracts/ModuleRegistry.sol

```diff
@@ -255,12 +279,14 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
     * @notice -> Only if Polymath enabled the feature.
     * @param _moduleFactory is the address of the module factory to be verified
     */
-    function unverifyModule(address _moduleFactory) external {
+    function unverifyModule(address _moduleFactory) external nonReentrant {
         // Can be called by the registry owner, the module factory, or the module factory owner
         bool isOwner = msg.sender == owner();
-        bool isFactoryOwner = msg.sender == IOwnable(_moduleFactory).owner();
         bool isFactory = msg.sender == _moduleFactory;
-        require(isOwner || isFactoryOwner || isFactory, "Not authorised");
+        bool isFactoryOwner = msg.sender == getAddressValue(Encoder.getKey("factoryOwner", _moduleFactory));
+        // Ordering (lazy evaluation) in below require statement is important. A moduleFactory should not be able
```

Comment:
This code comment isn't valid anymore, is it?

Reply:
True - was from an earlier iteration - removed.